data_gateway/packet_reader.py

```diff
@@ -413,17 +413,28 @@ def _parse_info_packet(self, information_type, payload):
                 logger.info("Microphone data erasing done")
             elif payload[0] == 3:
                 logger.info("Microphones started ")
+            return
 
-        elif information_type == "Cmd Decline":
+        if information_type == "Cmd Decline":
             reason_index = str(int.from_bytes(payload, self.config.endian, signed=False))
             logger.info("Command declined, %s", self.config.decline_reason[reason_index])
+            return
 
-        elif information_type == "Sleep State":
+        if information_type == "Sleep State":
             state_index = str(int.from_bytes(payload, self.config.endian, signed=False))
             logger.info("\n%s\n", self.config.sleep_state[state_index])
-            self.sleep = bool(int(state_index))
 
-        elif information_type == "Info Message":
+            if bool(int(state_index)):

```

Comment:
Can you explain why this has changed in the PR description?

Reply:
Done